includes/Admin/Admin_Page.php

```diff
@@ -47,8 +47,6 @@ public function add_hooks() {
 	 * Registers the admin page under the tools menu.
 	 *
 	 * @since n.e.x.t
-	 *
-	 * @return string The hook identifier for the admin page.
```

Comment:
This also needs to be added back. We are returning the hook, so it needs to be documented.